hack/update-codegen.sh

```diff
@@ -24,3 +24,10 @@ bash "${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client,informer,lister" \
   sigs.k8s.io/azuredisk-csi-driver/pkg/apis \
   azuredisk:v1beta1,v1beta2 \
   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt
+
```

Comment:
Let's put this in a separate script since the azuredisk CRDs and test CRDs won't need to be updated at the same time, e.g. `update-test-crds.sh`.